### ISSUE TITLE
[Bug] Prevent overwriting

### ISSUE THREAD

**Issue by Brilator** [nfdi4plants/arcCommander#47]
Adding a new item (e.g. investigation) with duplicate title / title already existing in isa.investigation.xlsx should not just overwrite the existing item. 

## Reproduce error
1. `arc i create --identifier "investigation_001"`
2. `arc i person register -l YourLastname -f YourFirstname -m YourMidInitials`
3. `arc i create --identifier "investigation_001"`
4. Person info gone 

## Suggestion 
Check for existing items. Respond with error "already exists" and / or suggest new name.

**Comment by HLWeil** [comment]
Good point, have you noticed this kind of missbehaviour in other cases too?

**Comment by Brilator** [comment]
Haven't tested, yet.

**Comment by Brilator** [comment]
Yes. Complete assay folder is also overwritten. 
```
arc a add -a TestAssay
echo 'bla bla test' > assays/TestAssay/README.md 
arc a add --assayidentifier "TestAssay"
cat assays/TestAssay/README.md
```
